src/utils/fetch-graphql.js

```diff
@@ -260,33 +280,36 @@ const handleFetchErrors = async ({
         }
          \n${genericError({ url })}`,
         { useVerboseStyle: true }
-      )
+      ),
+      CODES.Authentication
     )
   }
 
   const forbidden = e.message.includes(`Request failed with status code 403`)
 
   if (forbidden) {
-    reporter.panic(
+    structuredReporter.panic(
       formatLogMessage(
         `${e.message}\n\nThe GraphQL request was forbidden.\nIf you are using a security plugin like WordFence or a server firewall you may need to whitelist your IP address or adjust your firewall settings for your GraphQL endpoint.\n\n${errorContext}`
-      )
+      ),
+      CODES.Permissions
     )
   }
 
   const redirected = e.message.includes(`GraphQL request was redirected`)
 
   if (redirected) {
     await handleErrorOptions({ variables, query, pluginOptions, reporter })
-    reporter.panic(
+    structuredReporter.panic(
       formatLogMessage(
         `${e.message}\n\n${errorContext}\n\nThis can happen due to custom code or redirection plugins which redirect the request when a post is accessed.\nThis redirection code will need to be patched to not run during GraphQL requests.\n\nThat can be achieved by adding something like the following to your WP PHP code:\n
 if ( defined( 'GRAPHQL_REQUEST' ) && true === GRAPHQL_REQUEST ) {
   return examplePreventRedirect();
 }
 
 ${slackChannelSupportMessage}`
-      )
+      ),
+      CODES.CustomUserCode
```

Comment:
This should be WordPressFilters I think. The custom code that causes redirections is on the WP side and uses filters